truss/cli/cli.py

```diff
@@ -86,6 +86,10 @@
 
 is_humanfriendly_log_level = True
 
+# In the CLI flow, ignore any local ~/.aws/config files,
+# which can interfere with uploading the Truss to S3.
+os.environ["AWS_CONFIG_FILE"] = ""
```

Comment:
I don't like to have this long-range, stateful side effect from the top of the CLI file to where s3 clients are actually instantiated somewhere deep in the code base.

Esp. if we do pushes in a programmatic way and don't go via importing this file, it might lead to divergent behavior.

To ignore local config, could you provide appropriate instantiation arguments where s3 clients are created?

Reply:
I will double check this, but even if you add the credentials, it still pulls other settings from the config file (like the

```
s3 = 
  use_accelerate_endpoint = true
 ```

setting

Reply:
Such a crappy API. I already had to scratch my head that you get the client, by providing a string literal `"s3"` - like typing :shrug: 

But maybe it's still better to push down the "clearing of the env variable" to the same location to where the client is created.

Reply:
yep, this doesn't work :( Something that did work is

```
    session = boto3.Session(
        aws_access_key_id=credentials['aws_access_key_id'],
        aws_secret_access_key=credentials['aws_secret_access_key'],
        aws_session_token=credentials.get('aws_session_token')
    )
    
    s3_resource = session.resource("s3", config=config.Config(s3={"use_accelerate_endpoint": False}))
```

but this also doesn't feel great

Reply:
> But maybe it's still better to push down the "clearing of the env variable" to the same location to where the client is created.

I'll try something along these lines